src/aat/resources/features/F-201 - New Access Control Get Case/F-201_CT1.td.json

```diff
@@ -0,0 +1,81 @@
+{
+  "_guid_": "F-201_CT1",
+
+  "productName": "CCD Data Store",
```

Comment:
for all files using the /case-types/{ctid}/cases endpoint lets create and extend a base file like F-201_GetCaseBase.td.json and put the common lines of code here like the api details and request headers, then we can remove this common lines from all the files extending them